plugins/devtools/README.md

```diff
@@ -410,9 +410,21 @@ export const customDevToolsPage = <DevToolsPage />;
 
 The following sections outline the configuration for the DevTools plugin
 
+### Package Dependencies
+
+By default, only packages with names starting with `@backstage/` will be listed on the main "Info" tab. If you would like additional packages to be listed, you can specify the package prefixes in your `app-config.yaml`. For example, to include backstage plugins provided by the core application as well as `@roadiehq` and `@spotify`:
```

Comment:
Should we note that it only looks based on `startsWith` and does not support `RegExp`? Just thinking that might come up as something that gets asked about.

Also I'm wondering if we should remove the trailing slash from the example to make it a bit more clear. So `@backstage/` would be just `@backstage`.

Reply:
done and done